Documentation/operations/upgrade-warning.rst

```diff
@@ -4,6 +4,12 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. warning::
+
+   All Cilium components should run the same version for operation. Upgrading
+   just one of them (e.g., upgrading the agent without upgrading the operator)
+   could result in unexpected cluster behaviors.
```

Comment:
I wonder if we can just move text like this further down into the actual page rather than this upgrade-warning snippet which is included in multiple places. Perhaps here?

https://deploy-preview-17755--cilium-doc.netlify.app/operations/upgrade.html#upgrading-cilium

Then perhaps we can phrase it not as a warning, but more like an overview, something like:

```suggestion
   During normal cluster operations, all Cilium components should run the same version for operation. Upgrading
   just one of them (e.g., upgrading the agent without upgrading the operator)
   could result in unexpected cluster behavior. The following steps will describe how to upgrade all of the components from one stable release to a later stable release.
```

(This could be immediately above the second upgrade-warning include).

Reply:
I like your suggestion :) Having 2 warnings right in the beginning does seem a bit intimidating now that I look at it...